www/squid/src/opnsense/mvc/app/models/OPNsense/Proxy/Proxy.xml

```diff
@@ -405,6 +405,7 @@
                                 <Mask>/^([\t\n\v\f\r 0-9a-zA-Z.,_\x{00A0}-\x{FFFF}]){1,255}$/u</Mask>
                             </password>
                             <filter type="JsonKeyValueStoreField">
+                                <ConfigdPopulateAct>blacklist categories list</ConfigdPopulateAct>
```

Comment:
I don't think a configd action named `blacklist` exists.... easy to try using `configctl blacklist categories list` on a console. These `%s.index` files are created as a result from the download, triggered by `fetchACLs.py`:

https://github.com/opnsense/plugins/blob/30dd74260cec698f0221981fddbccb247489d095/www/squid/src/opnsense/service/conf/actions.d/actions_proxy.conf#L54-L70


https://github.com/opnsense/plugins/blob/30dd74260cec698f0221981fddbccb247489d095/www/squid/src/opnsense/scripts/proxy/fetchACLs.py#L357

Reply:
@abonanni it looks like a regression in https://github.com/opnsense/core/commit/525481c15ba2b7f3dd90418d43b2f943e8662511, previously a callout was not required. If you open a ticket in core, I'll take a look.